conf/openmetadata.yaml

```diff
@@ -113,6 +113,23 @@ database:
 migrationConfiguration:
   path: "./bootstrap/sql/mysql"
 
+# Authorizer Configuration
+authorizerConfiguration:
+  className: ${AUTHORIZER_CLASS_NAME:-org.openmetadata.catalog.security.NoopAuthorizer}
+  containerRequestFilter: ${AUTHORIZER_REQUEST_FILTER:-""}
```

Comment:
Should this be 


```
${AUTHORIZER_REQUEST_FILTER:-org.openmetadata.catalog.security.CatalogSecurityContextRequestFilter}
```

Reply:
Missed it, however it should be NoopFilter. Fixed it in latest commit